frontend/src/app/forms/steps/pictures-entry/pictures-entry.component.scss

```diff
@@ -0,0 +1,15 @@
+.custom-button {
+  flex-grow: 1;
+  flex-direction: column;
+  align-items: center;
+  display: flex;
+  justify-content: flex-end;
```

Comment:
Why do we need this as flex-end aligns the content at end? Also flex-grow: 1;
  flex-direction: column;
  align-items: center doesn't really needed now as there is only one element inside it.

Reply:
fixed this. (will upload it tomorrow morning)